benchmark/solkz/compositional_field/CMakeLists.txt

```diff
@@ -33,7 +33,16 @@ ENDIF ()
 DEAL_II_INITIALIZE_CACHED_VARIABLES()
 
 SET(TARGET "solkz_compositional_field")
+SET(TARGET_DEPENDENCY "solkz")
+
 PROJECT(${TARGET})
 
+INCLUDE_DIRECTORIES("..")
+
 ADD_LIBRARY(${TARGET} SHARED solkz_compositional_field.cc)
-ASPECT_SETUP_PLUGIN(${TARGET})	
+ADD_LIBRARY(${TARGET_DEPENDENCY} SHARED ../solkz.cc)
```

Comment:
The only fallback is that I have a relative path `../solkz.cc` and would like to change this to something along the lines of `${CMAKE_ASPECT_SOURCE_DIR}/benchmarks/solkz/solkz.cc` 

Does such a cmake variable exist? If not, we can always force the user to specify such a flag when running cmake. An additional flag with an informative cmake error message is one possible solution. Should I push forward with this in mind, @gassmoeller?